.github/workflows/release.yaml

```diff
@@ -199,10 +214,11 @@ jobs:
       - uses: baptiste0928/cargo-install@v2
         with:
           crate: cargo-release
-      - uses: richb-hanover/cargo@v1.1.0
-        with:
-          command: release
-          args: publish --no-confirm -x
+      # Currently, we can only check prql-ast which is not dependent other local crates with --dry-run.
+      # https://github.com/crate-ci/cargo-release/issues/691
+      - run:
+          cargo release publish --no-confirm ${{ inputs.dry-run && '--no-verify
```

Comment:
(I had thought that we wouldn't need the `--no-verify` if we were only doing `prql-ast`, but no big deal even if that is correct)

Reply:
> (I had thought that we wouldn't need the `--no-verify` if we were only doing `prql-ast`, but no big deal even if that is correct)

If this is not added, prql-ast will be built, which will take time.

Reply:
OK — I do think that's worth noting in a comment, so the next person knows why we have it. (Or is it fairly obvious and I was missing it? No offense taken if so, we can't explain every line of code... :) )

Reply:
I added a comment.